src/sui-link-item/index.jsx

```diff
@@ -51,7 +53,8 @@ export default class LinkItem extends Component {
           target={target}
           title={title}
           to={url}
-          onClick={handleClick}>
+          onClick={handleClick}
+          rel={nofollow ? 'nofollow' : null}>
```

Comment:
I think this would render an empty rel attribute if nofollow values false, so I would change this to:

```
...
{nofollow && rel="nofollow">
...
```
In order to avoid render an empty "rel" attribute.